### DISCUSSION TITLE
What is your favorite game please tell me.

### DISCUSSION THREAD

**Discussion by polarbearpppp** [NAIST-SE/AttractingContributors#1118] [2024 Attraction Contributions]
I really want to know your favorite game please share to me 
https://github.com/polarbearpppp/AttractingContributors-Games

**Comment by Ngochuy2137** [comment]
I have just contributed to your repo. Can you contribute to my repo please https://github.com/Ngochuy2137/AttractingContributors-fruits/tree/main ?

**Comment by Drobotsan** [comment]
Hi, I contribute to your project.
Please accept my pull request.

Would you be kind to contribute with my project :D
https://github.com/Drobotsan/Places-in-the-world

Discussion link: https://github.com/NAIST-SE/AttractingContributors/discussions/1140

**Comment by thapakorn604** [comment]
Hi, I have contributed to your project as ->https://github.com/polarbearpppp/AttractingContributors-Games/pull/7
Would you mind to create one to mine ? -> https://github.com/NAIST-SE/AttractingContributors/discussions/1115

**Comment by Marsall-k** [comment]
I have just made a PR on your Repo!
Could you contribute to mine?
https://github.com/Marsall-k/AttractingContributors-Pokemon
